templates/components/form/form.jsx

```diff
@@ -55,9 +57,13 @@ var Form = React.createClass({
       valid_username: true,
       valid_password: true,
       valid_feedback: true,
-      valid_email: true
+      valid_email: true,
+      checkedUsername: false
     };
   },
+  setAPIState: function(data) {
```

Comment:
Doesn't look like you're using this anywhere now. Same with the setAPIState event.